ftw/labels/profiles.zcml

```diff
@@ -39,4 +39,12 @@
             />
     </genericsetup:upgradeSteps>
 
+    <genericsetup:upgradeStep
+        title="Go to ftw 1.3.1"
+        description="Update jar storage and ILabelSupport storage"
+        source="2000"
+        destination="2001"
+        handler="ftw.labels.migrations.migrate.migrate_to_2001"
```

Comment:
Update this to `handler="ftw.labels.upgrades.upgrade_to_2001.upgrade_to_2001"` after the other renames.